doc/source/ray-core/actors/fault-tolerance.rst

```diff
@@ -3,7 +3,7 @@
 Fault Tolerance
 ===============
 
-Ray will automatically restart actors that crash unexpectedly.
+Ray will automatically restart actors that crash unexpectedly (not including application failures raised in the ``__init__`` function).
```

Comment:
I think a better description of crash would suffice here. "crash" means that the actor should die. raise ValueError does not crash the algorithm the same way that `sys.exit()` does. @gjoliver and I confirmed that if in `__init__()` we do `sys.exit()` it will try to recover it as expected.

Reply:
Ah, good point, so the max_restarts does get respected if the actor process dies (or crash)

Reply:
Yes